tweepy/asynchronous/pagination.py

```diff
@@ -4,13 +4,24 @@
 
 from math import inf
 
+import aiohttp
+
+from tweepy.client import Response
+
 
 class AsyncPaginator:
     """:class:`AsyncPaginator` can be used to paginate for any
     :class:`AsyncClient` methods that support pagination
 
     .. versionadded:: 4.11
 
+    .. note::
+
+        When the returned response from the method being passed is of type
+        :class:`aiohttp.ClientResponse`, it will be deserialized in order to parse
+        the pagination tokens, likely negating any potential performance
+        benefits from using a :class:`aiohttp.ClientResponse` return type.
```

Comment:
```suggestion
    .. note::
        When the returned response from the method being passed is of type
        :class:`aiohttp.ClientResponse`, it will be deserialized in order to
        parse the pagination tokens, likely negating any potential performance
        benefits from using a :class:`aiohttp.ClientResponse` return type.

    .. versionadded:: 4.11
```

Reply:
The line length adjustment here still applies.

Reply:
Fixed. 79 limit? 😂 you could add an `autopep8` linter or `yapf`.